library/src/main/resources/SAMLSpecRefMessage.properties

```diff
@@ -302,6 +302,22 @@ SAMLCore_8_3_6_b=[T]he content of the element is the identifier of an entity tha
   SAML profiles (such as a service provider supporting the browser SSO profile)...The syntax of \
   such an identifier is a URI of not more than 1024 characters in length.
 
+SAMLCore_8_3_7_a=Persistent name identifier values MUST NOT exceed a length of 256 characters.
+
+SAMLCore_8_3_7_b=The element's NameQualifier attribute, if present, MUST contain the unique \
```

Comment:
❗️ We should add this to `NotTested`.

In the reasoning we should note that we can't tell if a message was generated from another SAML entity.

Reply:
Not sure what you mean? It says that the `NameQualifier` needs to have the IdP's entity ID.